webui/react/src/utils/cluster.ts

```diff
@@ -12,7 +12,11 @@ export const getSlotContainerStates = (
     .flatMap((agent) => {
       const arr: ResourceState[] = Object.entries(agent.slotStats.typeStats ?? {})
         .filter(([type]) => {
-          return resourceType === ResourceType.ALL || type === `TYPE_${resourceType}`;
+          return (
+            resourceType === ResourceType.ALL ||
+            type === `TYPE_${resourceType}` ||
+            resourceType === 'UNSPECIFIED'
+          );
```

Comment:
prefer `resourceType === ResourceType.UNSPECIFIED` over raw string.

but also, I think we could just replace with `isDeviceType` from `types.tsx`:

```
return !isDeviceType(resourceType) || type === `TYPE_${resourceType}`
```

Reply:
done